app/integration/component-proxy/src/main/java/io/syndesis/integration/component/proxy/ComponentDefinition.java

```diff
@@ -98,6 +115,7 @@
         Optional<String> getPrefix();
         Optional<String> getMultiValue();
         Optional<String> getEnumValues();
+        Optional<List<PropertyRelation>> getRelation();
```

Comment:
Making a collection optional is ugly, getRelation should return an empty collection by default